### PR TITLE
Components initial move

### PR DESCRIPTION
**Pull request by heliocastro**

### CODE REVIEW

**Review by KoukiHama — APPROVED** [review]

File: src/app/[locale]/licenses/page.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    if (fetchData !== null) {
+        data = fetchData.map((item) => [
+            _(<Link href={item._links.self.href}>{item._links.self.href.split('/').pop()}</Link>),
+            item.fullName,
+            _(<center>{item.checked ? <Check2Circle color='#287d3c' size='16' /> : <XCircle color='red' />}</center>),
+        ])
+    }
+
+    const doSearch = (event: React.KeyboardEvent<HTMLInputElement>) => {
+        console.log(event.currentTarget.value)
```
Comment (hoangnt2) [inline src/app/[locale]/licenses/page.tsx:63]: Remove this console.log

**Comment by hoangnt2** [comment]
@heliocastro
Could you add the messages for another languages (vi, zh, ja)

**Comment by heliocastro** [comment]
> @heliocastro Could you add the messages for another languages (vi, zh, ja)

Did it better.
Simple utility in utils to track and add all mesages included in en.json and marked as NOT TRANSLATED
I sorted the strings as well.

In the future we can make this automatised and **maybe** add automatic translation with some translator API ?

**Comment by hoangnt2** [comment]
Code looks good

**Review by hoangnt2 — APPROVED** [review]

**Pull request merged**